pytorch_lightning/plugins/precision/hpu.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (C) 2021 Habana Labs, Ltd. an Intel Company
+# All Rights Reserved.
+#
+# Unauthorized copying of this file or any element(s) within it, via any medium
+# is strictly prohibited.
+# This file contains Habana Labs, Ltd. proprietary and confidential information
+# and is subject to the confidentiality and license agreements under which it
+# was provided.
+#
+
```

Comment:
Is there any reason why this Habana copyright was included in this file? This is highly unusual.

Reply:
I believe this needs to be updated as commented in https://github.com/PyTorchLightning/pytorch-lightning/pull/10404#discussion_r744560909 by @tchaton.